package.json

```diff
@@ -728,7 +728,7 @@
                     "dark": "resources/dark/variable_explorer.svg"
                 },
                 "category": "Jupyter",
-                "enablement": "!jupyter.variableViewVisible"
+                "enablement": "jupyter.isvscodenotebookactive && !jupyter.variableViewVisible"
```

Comment:
You can use this instead `notebookViewType == jupyter-notebook`
I.e. we don't need a new context variable.

Reply:
@DonJayamanne not sure if you have a thought on this. But this doesn't seem to be true. In particular on the view enablement notebookViewType == jupyter-notebook doesn't work and the view never shows.

Reply:
VSCode bug?

Reply:
Yeah, I'd say its a VS Code bug, could be related to the same issue where toolbar icons do not appear until you click on the notebook.